intro_to_R_tidyverse/01-intro_to_r.Rmd

```diff
@@ -8,15 +8,31 @@ output:
 
 **CCDL 2019**
 
-### This RMarkdown takes you through the basics of R  
+### This [R Markdown](http://rmarkdown.rstudio.com) Notebook takes you through the basics of R  
 
 *For more comprehensive tutorials:*  
   - For a tutorial on R, RStudio and R Markdown: https://ismayc.github.io/rbasics-book/  
-  - [Datacamp](https://www.datacamp.com/) has lots of good tutorials for R and the intro ones are free.       
   - For a lot of handy R cheatsheets: https://www.rstudio.com/resources/cheatsheets/  
   - For more guidance all the things you can do with R Notebooks:    
     https://bookdown.org/yihui/rmarkdown/notebook.html  
-    
+
+#### A brief tutorial on how to use R Markdowns:
+
```

Comment:
Could we preface this by saying that R Markdowns can have sections of text and chunks of code?

Reply:
That will be covered in the slides before we get to this.